test/WebJobs.Script.Tests.Integration/Management/FunctionsSyncManagerTests.cs

```diff
@@ -290,7 +290,10 @@ public async Task TrySyncTriggers_CheckHash_PostsExpectedContent()
             using (var env = new TestScopedEnvironmentVariable(_vars))
             {
                 var hashBlob = await _functionsSyncManager.GetHashBlobAsync();
-                await hashBlob.DeleteIfExistsAsync();
+                if (hashBlob != null)
+                {
+                    await hashBlob.DeleteIfExistsAsync();
+                }
```

Comment:
was there a behavior change here?

Reply:
I think I made this change a while back as a result of some first-run null-ref test failures. I don't think there is any behavior change here, just a slightly more robust test.